src/server/hook_func.c

```diff
@@ -3857,7 +3857,7 @@ process_hooks(struct batch_request *preq, char *hook_msg, size_t msg_len,
 		}
 		rc = server_process_hooks(preq->rq_type, preq->rq_user, preq->rq_host, phook,
 					  hook_event, pjob, &req_ptr, hook_msg, msg_len, pyinter_func, &num_run);
-		if ((rc == 0) || (rc == 1))
+		if (rc == 0)
```

Comment:
@vchlum thanks for fixing this issue. I should have tested it with multiple hooks of same kind when I made this change.
I think you might still want to check for rc to be "-1" and return because when it is "-1" there might be an internal error and you would not want to continue further.